src/main/java/com/breskul/bring/AnnotationConfigApplicationContext.java

```diff
@@ -36,31 +39,117 @@ public AnnotationConfigApplicationContext(String packageName) {
     }
 
     /**
-     * Scanning a package for classes annotated by {@link Component}.
-     * It creates an instances of such classes and put them into context.
+     * <h3>Scanning a package for classes annotated by {@link Component}.</h3>
+     * <p>It creates an instances of such classes and put them into context.</p>
      */
     private void initiateContext() {
         Reflections reflections = new Reflections(packagePath, Scanners.TypesAnnotated);
+        Set<Class<?>> futureComponents = reflections.getTypesAnnotatedWith(Component.class);
```

Comment:
`futureComponents `name does not describe real name of type of objects with which we work. Probably better simplify name to `classes`